test/scripts/run-travis-push-or-pr

```diff
@@ -2,11 +2,16 @@
 set -e
 
 if [ "$TRAVIS_BRANCH" == "master" ]; then
-    # this is a "pr" job for travis-ci, run the full test suite, one browser at a time
+    # this is a "pr" job for travis-ci, run the full saucelabs suite
     if [ "$#" -eq "0" ]; then
+        echo "Testing Edge"
+        npm run test:regression -- --browsers=edge
+        echo "Testing IE"
+        npm run test:regression -- --browsers=ie
         echo "Testing Chrome"
         npm run test:regression -- --browsers=chrome
-
+        echo "Testing Safari"
+        npm run test:regression -- --browsers=safari
         echo "Testing Firefox"
         npm run test:regression -- --browsers=ff
```

Comment:
Do we need to explicitly run regression tests for each browser individually?

Wouldn't running `npm run test:regression` (without the `--browsers` option) default to running the full suite?

Reply:
npm run test:regression We can get this as well. Just how we did for components. For all components we use:  npm run test:regression --browsers=ff,chrome,safari,ie,edge

Reply:
Talking with @shaleen76, the reason we have separate commands is to implement a sort of "parallel" run on Saucelabs. If this is true, I'm good with it.